src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -4,6 +4,7 @@
     <Description>PowerShell Core's System.Management.Automation project</Description>
     <NoWarn>$(NoWarn);CS1570;CS1734</NoWarn>
     <AssemblyName>System.Management.Automation</AssemblyName>
+    <LangVersion>7.2</LangVersion>
```

Comment:
Do we really need this? Currently we use default and seems the default is 7.2.

Reply:
Span was not found and from what I could find, currently we need to opt into 7.2 features.  Perhaps it will be default once dotnetcore2.1 GAs.

Reply:
@SteveL-MSFT I found in [docs](https://docs.microsoft.com/en-us/dotnet/csharp/reference-semantics-with-value-types) that we could use `Latest` - if this works maybe we expand this on all proj files?

/cc @daxian-dbw

Reply:
The `<LanguageVersion>` tag should be put in `PowerShell.Common.props`, I think.

Reply:
I opened the tracking Issue #6547.